Update event-segmentation-choose-measurement.md (#747)
amplitude/amplitude-docs: content/collections/event-segmentation/en/event-segmentation-choose-measurement.md

## Formula

> In an Event Segmentation chart, you can write formulas that Amplitude will apply to the events you've included in your analysis. To read more about each formula and see some examples of use cases, see our [Custom Formulas](/docs/analytics/charts/event-segmentation/event-segmentation-custom-formulas) article.

Reviewer: ```suggestion
In an Event Segmentation chart, you can write formulas for Amplitude to apply to the events you include in your analysis. To read more about each formula and see some examples of use cases, see [Custom Formulas](/docs/analytics/charts/event-segmentation/event-segmentation-custom-formulas).
```
